src/components/Notifications/NotificationEmailCard/NotificationEmailCard.tsx

```diff
@@ -91,9 +94,15 @@ function NotificationEmailCard(props: Props) {
           <Title variant="h6" data-testid={NOTIFICATION_EMAIL_CARD_TITLE_TEST_ID}>
             {t('settings.notifications.email.title')}
 
-            {(unconfirmedEmail || hasConfirmEmail) && (
-              <SpanUnconfirmedEmail confirmed={hasConfirmEmail} data-testid={NOTIFICATION_EMAIL_CARD_UNCONFIRMED_TEST_ID}>
-                {hasConfirmEmail ? t('settings.notifications.email.confirmed') : t('settings.notifications.email.pending_approval')}
+            {unconfirmedEmail && !hasConfirmEmail && (
+              <SpanUnconfirmedEmail data-testid={NOTIFICATION_EMAIL_CARD_UNCONFIRMED_TEST_ID}>
+                {t('settings.notifications.email.pending_approval')}
+              </SpanUnconfirmedEmail>
+            )}
+
+            {hasConfirmEmail && !unconfirmedEmail && (
+              <SpanUnconfirmedEmail confirmed data-testid={NOTIFICATION_EMAIL_CARD_UNCONFIRMED_TEST_ID}>
+                {t('settings.notifications.email.confirmed')}
```

Comment:
As the component is the same, what if we add the conditionals to the text that needs to be rendered?